all/build.gradle

```diff
@@ -14,6 +14,10 @@ buildscript {
 def subprojects = [
   project(':instrumentation-java-core'),
   project(':instrumentation-java-core-impl'),
+  project(':instrumentation-java-core-impl-android'),
```

Comment:
For the moment I would not put android/java7/java8 here. We should probably have 3 code coverage instead of all (java7, java8, android) to avoid the duplicate class names issue.